zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreter.java

```diff
@@ -52,6 +52,7 @@
   private String interpreterRunner;
   private String interpreterPath;
   private String localRepoPath;
+  private String mvnRepoPath;
```

Comment:
While `Booter.java` read environment variable and jvm property directly and `SparkInterpreter.java` does not need to read `zeppelin.interpreter.dep.mvnRepo`, we don't need to change `RemoteInterpreter.java`.

`RemoteInterpreterManagedProcess.java`
`RemoteAngularObjectTest.java`
`RemoteInterpreterOutputTestStream.java`
`RemoteInterpreterProcessTest.java`
`RemoteInterpreterTest.java`
`DistributedResourcePoolTest.java`
`RemoteSchedulerTest.java`
`InterpreterFactory.java`

as well.